src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -237,16 +238,17 @@ protected internal void SetHtmlContent(string html)
 			Uno.UI.Xaml.WindowManagerInterop.SetContentHtml(HtmlId, html);
 		}
 
-		partial void EnsureClip(Rect rect)
+		partial void ApplyNativeClip(Rect rect)
 		{
+
 			if (rect.IsEmpty)
 			{
 				SetStyle("clip", "");
 				return;
 			}
 
-			var width = double.IsInfinity(rect.Width) ? 100000.0f : rect.Width;
-			var height = double.IsInfinity(rect.Height) ? 100000.0f : rect.Height;
+			var width = double.IsInfinity(rect.Width) ? 1000000.0f : rect.Width;
```

Comment:
I don't remember why there's this magic value, we may want to comment on that and make it a constant.